ansible/roles/linux-ansible-init/templates/ansible-init.sh.j2

```diff
@@ -4,6 +4,11 @@ set -exo pipefail
 
 # This script executes the playbooks from /etc/ansible-init/playbooks in lexicographical order
 
+# Make sure we operate in our virtual environment
+source {{ ansible_init_venv_path }}/bin/activate
+
 for file in $(find /etc/ansible-init/playbooks -maxdepth 1 -type f -name '*.yml' | sort -n); do
     ansible-playbook --connection local --inventory 127.0.0.1, "$file"
```

Comment:
It is sufficient to just point to the `ansible-playbook` command in the venv, e.g. `{{ ansible_init_venv_path }}/bin/ansible-playbook`. We don't need to activate/deactivate.

Reply:
Done